test/cli/detect_breakends_cli_test.cpp

```diff
@@ -35,15 +35,24 @@ TEST_F(cli_test, fail_no_argument)
 TEST_F(cli_test, with_arguments)
 {
     cli_test_result result = execute_app("detect_breakends",
-                                         data("converted_bam_shorted.sam"),
+                                         data("simulated.minimap2.hg19.coordsorted_cutoff.sam"),
                                          "detect_breakends_insertion_file_out.fasta");
     std::string expected
     {
-        "Reference\tchr9\t70103073\tForward\tReference\tchr9\t70103147\tForward\tm13802/6999/CCS\n"
+        "Reference\tchr21\t41972616\tForward\tRead \t0\t2294\tForward\tm2257/8161/CCS\n"
+        "Reference\tchr21\t41972616\tReverse\tRead \t0\t3975\tReverse\tm2257/8161/CCS\n"
+        "Reference\tchr22\t17458417\tForward\tReference\tchr21\t41972615\tForward\tm41327/11677/CCS\n"
+        "Reference\tchr22\t17458418\tForward\tReference\tchr21\t41972616\tForward\tm21263/13017/CCS\n"
+        "Reference\tchr22\t17458418\tForward\tReference\tchr21\t41972616\tForward\tm38637/7161/CCS\n"
     };
     std::string expected_err
```

Comment:
Is there already an issue for why this writes to `err` ?

Reply:
Jepp: https://github.com/seqan/igenvar/issues/23